build.psm1

```diff
@@ -1971,7 +1971,7 @@ function Start-DevPowerShell {
             Remove-Item env:DEVPATH
         }
 
-        if ($ZapDisable) {
+        if (Get-Variable -Name ZapDisable -ValueOnly -ErrorAction SilentlyContinue) {
```

Comment:
I think we should remove the if at all.

Reply:
just the if statement or the line inside too? Leaving the line `Remove-Item env:COMPLUS_ZapDisable` will cause Start-DevPowerShell to throw this error on exit

```> Line |
> 1974 |          Remove-Item env:COMPLUS_ZapDisable
>      |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>      | Cannot find path 'Env:\COMPLUS_ZapDisable' because it does not exist.
>

Reply:
All the `if` block.

(NGEN was used only for building PowerShell Native (aka Windows PowerShell). We removed this a long time ago.)